seventeenth_hw/page_objects/manager_application_form/manager_application_form.py

```diff
@@ -0,0 +1,70 @@
+from seventeenth_hw.page_objects.manager_application_form.manager_application_form_locators import \
+    ManagerApplicationFormLocators
+from seventeenth_hw.utilities.web_ui.base_page import BasePage
+
+
+class ManagerApplicationForm(BasePage):
+    def __init__(self, browser):
+        super().__init__(browser)
+        self.__manager_form_locator = ManagerApplicationFormLocators()
+
+    __phone_onfocus_attribute = "onfocus"
+
+    def is_form_visible(self):
+        return self._wait_until_element_visible(self.__manager_form_locator.popup)
+
+    def is_phone_onfocus_exist(self):
+        return self._find_attribute_in_locator(self.__manager_form_locator.phone_input, self.__phone_onfocus_attribute)
+
+    def click_exit_button(self):
+        return self._click(self.__manager_form_locator.exit_button)
```

Comment:
![image](https://user-images.githubusercontent.com/25909810/229343448-9f1cc0dc-9efa-4a02-8a01-8a4b7a9572c3.png)
It returns None, so there is no need for return